wiki/Mascots/en.md

```diff
@@ -14,11 +14,11 @@ pippi, stylized with a lowercase "p", is the osu!standard mascot that joined on
 
 ### ![osu!catch icon](/wiki/shared/mode/catch.png) Yuzu
 
-<!-- *For the news post, see: [Meet Yuzu](https://osu.ppy.sh/home/news/89483664163)* -->
+*For the news post, see: [Meet Yuzu](https://osu.ppy.sh/home/news/2014-06-21-meet-yuzu)*
```

Comment:
Also add rework newspost and maybe update description with the changes.
```suggestion
*For the news post, see: [Meet Yuzu!](https://osu.ppy.sh/home/news/2014-06-21-meet-yuzu)*
```

Reply:
Alright, I'll add in the link to the "Introducing Yuzu's New Look" news post and add a note in the description that the current design was by Thievley. I'll try to rewrite the descriptions another time, unless someone else wants to do it or does it before I do.